internal/commands/build.go

```diff
@@ -128,3 +157,21 @@ func addEnvVar(env map[string]string, item string) map[string]string {
 	}
 	return env
 }
+
+func readProjectDescriptor(appPath string, descriptorPath string) (project.Descriptor, error) {
+	var fullDescriptorPath string
+	if appPath != "" {
+		fullDescriptorPath = filepath.Join(appPath, descriptorPath)
+	} else {
+		fullDescriptorPath = descriptorPath
+	}
+
+	if _, err := os.Stat(fullDescriptorPath); !os.IsNotExist(err) {
+		return project.ReadProjectDescriptor(fullDescriptorPath)
+	}
+	return project.Descriptor{}, nil
+}
+
+func parseProjectToml(appPath string, descriptorPath string) (project.Descriptor, error) {
+	return readProjectDescriptor(appPath, descriptorPath)
```

Comment:
This indirection seems unnecessary. The functionality of `readProjectDescriptor` could just live here.

Reply:
Good point!